{"uid":"d55cb27c19c1ee02","name":"test_redeploy_charm_same_model_after_forcing_removal","fullName":"tests.integration.test_charm#test_redeploy_charm_same_model_after_forcing_removal","historyId":"ca9aac0977186ec16e8b66c76fd09de6","time":{"start":1747879516713,"stop":1747880527472,"duration":1010759},"description":"Redeploy the charm in the same model to test that it works after a forceful removal.","descriptionHtml":"<p>Redeploy the charm in the same model to test that it works after a forceful removal.</p>\n","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 898, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_charm.py\", line 446, in test_redeploy_charm_same_model_after_forcing_removal\n    await ops_test.model.wait_for_idle(\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3250, in wait_for_idle\n    raise asyncio.TimeoutError(\nTimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1747877408709,"stop":1747877408710,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1747877408710,"stop":1747877408710,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1747877408710,"stop":1747877408710,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1747877408710,"stop":1747877408945,"duration":235},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1747879516713,"stop":1747879516713,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Redeploy the charm in the same model to test that it works after a forceful removal.","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 898, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_charm.py\", line 446, in test_redeploy_charm_same_model_after_forcing_removal\n    await ops_test.model.wait_for_idle(\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3250, in wait_for_idle\n    raise asyncio.TimeoutError(\nTimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","steps":[],"attachments":[{"uid":"7f70f2309bce02d9","name":"log","source":"7f70f2309bce02d9.txt","type":"text/plain","size":8236}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1747880527478,"stop":1747880527478,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1747880657359,"stop":1747880657703,"duration":344},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1747880657704,"stop":1747880657704,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1747880657704,"stop":1747880657704,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1747880657704,"stop":1747880657704,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_charm"},{"name":"host","value":"fv-az1308-193"},{"name":"thread","value":"19175-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_charm"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"4d76826a9ddf05cf","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":3,"broken":36,"skipped":2,"passed":302,"unknown":4,"total":347},"items":[{"uid":"b95fd1e67b668787","reportUrl":"../4065//#testresult/b95fd1e67b668787","status":"passed","time":{"start":1747793045895,"stop":1747793373934,"duration":328039}},{"uid":"9184f4f5a8a46672","reportUrl":"../4058//#testresult/9184f4f5a8a46672","status":"passed","time":{"start":1747707491720,"stop":1747707824756,"duration":333036}},{"uid":"a54eae1246f1d542","reportUrl":"../4057//#testresult/a54eae1246f1d542","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","time":{"start":1747620205986,"stop":1747621215858,"duration":1009872}},{"uid":"9926293ef642b3a0","reportUrl":"../4056//#testresult/9926293ef642b3a0","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1747531874076,"stop":1747531874076,"duration":0}},{"uid":"466262486acca7c4","reportUrl":"../4055//#testresult/466262486acca7c4","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","time":{"start":1747447425040,"stop":1747448433428,"duration":1008388}},{"uid":"3727dd4ce772808b","reportUrl":"../4054//#testresult/3727dd4ce772808b","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","time":{"start":1747360959316,"stop":1747361967812,"duration":1008496}},{"uid":"37ae2a17dd2c9b0e","reportUrl":"../4053//#testresult/37ae2a17dd2c9b0e","status":"passed","time":{"start":1747274665775,"stop":1747275052273,"duration":386498}},{"uid":"d8d9059e4e710f62","reportUrl":"../4052//#testresult/d8d9059e4e710f62","status":"passed","time":{"start":1747188216279,"stop":1747188551237,"duration":334958}},{"uid":"d17d60cee464613a","reportUrl":"../4047//#testresult/d17d60cee464613a","status":"passed","time":{"start":1747101816884,"stop":1747102193554,"duration":376670}},{"uid":"4fbc768ff8bea174","reportUrl":"../4046//#testresult/4fbc768ff8bea174","status":"passed","time":{"start":1747015490420,"stop":1747015857475,"duration":367055}},{"uid":"1e8957f6b2ec7123","reportUrl":"../4045//#testresult/1e8957f6b2ec7123","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","time":{"start":1746928979547,"stop":1746929988806,"duration":1009259}},{"uid":"7110567e0aa3368b","reportUrl":"../4044//#testresult/7110567e0aa3368b","status":"passed","time":{"start":1746842466710,"stop":1746842816602,"duration":349892}},{"uid":"99c0ed8f2ec5c497","reportUrl":"../4043//#testresult/99c0ed8f2ec5c497","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","time":{"start":1746756205641,"stop":1746757215088,"duration":1009447}},{"uid":"13fb5bc3bbf52bb9","reportUrl":"../4040//#testresult/13fb5bc3bbf52bb9","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","time":{"start":1746670277483,"stop":1746671287594,"duration":1010111}},{"uid":"49eea03aa8c505d3","reportUrl":"../4030//#testresult/49eea03aa8c505d3","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","time":{"start":1746583286653,"stop":1746584296032,"duration":1009379}},{"uid":"78dd4438a564f53e","reportUrl":"../4009//#testresult/78dd4438a564f53e","status":"passed","time":{"start":1746497020138,"stop":1746497309334,"duration":289196}},{"uid":"a9abed3298615f67","reportUrl":"../4004//#testresult/a9abed3298615f67","status":"passed","time":{"start":1746410517119,"stop":1746410802092,"duration":284973}},{"uid":"34da2244feeef268","reportUrl":"../4000//#testresult/34da2244feeef268","status":"passed","time":{"start":1746324117000,"stop":1746324401572,"duration":284572}},{"uid":"985c21e1db047c31","reportUrl":"../3999//#testresult/985c21e1db047c31","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\n  postgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent","time":{"start":1746237980160,"stop":1746238989290,"duration":1009130}},{"uid":"5be6f3c40b909d6f","reportUrl":"../3995//#testresult/5be6f3c40b909d6f","status":"passed","time":{"start":1746151255087,"stop":1746151550888,"duration":295801}}]},"tags":["asyncio"]},"source":"d55cb27c19c1ee02.json","parameterValues":[]}